linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -1239,6 +1243,17 @@ impl NodeService {
         Ok(bytecode_id.with_abi())
     }
 
+    pub async fn query_committees(&self, chain_id: &ChainId) -> Result<BTreeMap<Epoch, Committee>> {
+        let query = format!(
+            "query {{ chain(chainId:\"{chain_id}\") {{
+                executionState {{ system {{ committees }} }}
+            }} }}"
+        );
```

Comment:
I wish this could be typesafe. Hardcoding strings into these queries had caused a lot of debugging headaches in the past.

Reply:
I agree. But I just moved this code, and we have lots of strings like this, so I'd like to not address that in this PR.